Modify README (#62)
gitpod-io/spring-petclinic: readme.md

# Spring PetClinic Sample application [![setup automated](https://img.shields.io/badge/Gitpod-ready_to_code-orange?logo=gitpod)](https://gitpod.io/from-referrer/)

Start coding in a [ready-to-code development environment](https://www.gitpod.io):
[Petclinic](https://github.com/spring-projects/spring-petclinic) is a [Spring Boot](https://spring.io/guides/gs/spring-boot/) application built using [Maven](https://spring.io/guides/gs/maven/).

<a href="https://gitpod.io/from-referrer/" style="padding: 10px;">
    <img src="https://gitpod.io/button/open-in-gitpod.svg" width="150" alt="Push" align="center">

> </a>

Reviewer: **Question:** Do we want to move this CTA down, seems logical to keep this up top as the big CTA? 

WDYT?

 - [Get Started With Your Own Project](#get-started-with-your-own-project)
 - [Community support](#community-support)

## Understanding the Spring Petclinic application with a few diagrams
<a href="https://speakerdeck.com/michaelisvy/spring-petclinic-sample-application">See the presentation here</a>

<img width="1042" alt="petclinic-screenshot" src="https://cloud.githubusercontent.com/assets/838318/19727082/2aee6d6c-9b8e-11e6-81fe-e889a5ddfded.png">

## Run petclinic on Gitpod

Start coding in a [ready-to-code development environment](https://www.gitpod.io)

> Setup Gitpod integration with Jetbrains Gateway using the instructions given in this [doc](https://www.gitpod.io/docs/ides-and-editors/intellij).

Reviewer: **Suggestion:** Refactor sentence to have the link be [title of the thing you're linking to] as opposed to [this]

```suggestion
Configure [IntelliJ in Gitpod](https://www.gitpod.io/docs/ides-and-editors/intellij) with [Jetbrains Gateway](https://www.gitpod.io/docs/ides-and-editors/jetbrains-gateway).
```

> Click the button below to start a new workspace on Gitpod which will automatically open the workspace in IntelliJ IDEA through JetBrains Gateway.  

Reviewer: **Thought:** Not sure if we should re-word this somehow...

1. First-time users are presented with an IDE choice (so will have the option for IntelliJ)
2. Second-time users and Gitpod will only open if the user has selected IntelliJ before. 

The latter might be quite common as VS Code Browser is the default, and we only ask on the first workspace. The user then has to ensure that they set their global preference for subsequent workspaces. Until we have hardcode-able URL's to ensure that specific IDEs are opened having this info in the readme might be brittle / get outdated quite quickly. 🤔

> <a href="https://gitpod.io/from-referrer/" style="padding: 10px;">

Reviewer 2: ```suggestion
<a href="https://gitpod.io/#https://github.com/gitpod-io/spring-petclinic" style="padding: 10px;">
```

    <img src="https://gitpod.io/button/open-in-gitpod.svg" width="150" alt="Push" align="center">
</a>

> Or you can run it from Maven directly using the Spring Boot Maven plugin. If you do this it will pick up changes that you make in the project immediately (changes to Java source files require a compile as well - most people use an IDE for this):
> 
> ```
> ./mvnw spring-boot:run
> ```

Reviewer: ```suggestion
```

Should we remove this section, not quite sure what it's adding? The phrasing makes it read like it's an optional way of running the quick start which is somewhat confusing to me.

## Get Started With Your Own Project

> To get started with Spring Boot on Gitpod, add a [`.gitpod.yml`](./.gitpod.yml) file which contains the configuration to improve the developer experience on Gitpod. To learn more, please see the [Getting Started](https://www.gitpod.io/docs/getting-started) documentation.

Reviewer: **Suggestion:** Swap to the new java doc? It'd be nice if there were spring specific info in there too, but we can add soon!

**Thought:** There are lots of configurations in Gitpod outside of the `.gitpod.yml`, I like the idea of giving a concrete next action, but wonder if we're better off just linking to the docs here. 

```suggestion
To get started with Spring Boot and Java on Gitpod, see the [Java Language Guide](https://www.gitpod.io/docs/languages/java).
```
